ClinGen/clincoded
# Connects to #1441. Connects to #1638. Connects to #1718. Connects to #1741.

**Technical notes:**
Testing #1718 requires production data to be loaded into an instance.

**Steps to test #1441:**
1. Log into the curation interfaces and click on the **Help** nav button in the top nav bar.
2. Expect to see the **Standard Operating Procedures** link in the **Help** dropdown menu.
3. Click on the **Standard Operating Procedures** link and expect to open a new window/tap loading a page from https://www.clinicalgenome.org/curation-activities/gene-disease-validity/educational-and-training-materials/standard-operating-procedures/.

**Steps to test #1638:**
1. Go to the VCI and enter ClinVar ID **17000** in the variant selection interface.
2. Proceed to the VCI evidence view and click on the **Interpretation +** button to start an interpretation.
3. Click on the **Variant Type** tab and then the **Loss of Function** sub-tab.
4. Expect to see a **Sequence Variant Interpretation (SVI) Working Group guidance** link below the **PVS1** criteria evaluation section.
5. Click on the link and expect to open a new window/tab loading a page from https://www.clinicalgenome.org/working-groups/sequence-variant-interpretation/.
6. Now click on the **Case/Segregation** tab and also expect to see a **Sequence Variant Interpretation (SVI) Working Group guidance** link below the **PM6/PS2** criteria evaluation section.
7. Clicking on this link should produce the same behavior as previously described.

**Steps to test #1718:**
1. From the **Dashboard** page, click on the **View list of all Gene-Disease Records** link.
2. On the subsequent _All Gene-Disease Records_ page, expect to see the _Loading..._ spinner appearing first and shortly after the list of GDMs are rendered on the page.
3. In the upper-right text input field, enter keyword _Autism_ and expect to see a filtered list of GDMs associated with the **Autism and Intellectual Disability EP** affiliation.
4. Return to the **Dashboard** page, click on the **View list of all Variant Interpretations** link.
5. On the subsequent _All Interpretations_ page, expect to see the same _Loading..._ spinner appearing first and the 3-4 seconds later the list of Interpretations are rendered on the page.
6. Similarly via the upper-right text input field, expect to be able to filter the list by variant title/ID, disease term/ID, or submitter name/affiliation (e.g. **PTEN EP**).

**Steps to test #1741:**
1. SSH into the instance and inspect the _ElasticSearch_ log as well as the _Apache Error_ log.
2. Expect to see the logs free of _Memory usage exceeds limit_ or _memlimit_ errors.
3. Also expect to see the all GDMs and Interpretations pages loading faster.
